frontend/epfl-memento/templates/includes/card-info.inc.php

```diff
@@ -1,45 +1,65 @@
-    $markup .= '<span>' . esc_html($start_time) . '</span>';
-    $markup .= '<span>' . esc_html($end_time) . '</span>';
+if ($event->start_date == $event->end_date){
+    $markup .= '<svg class="icon feather" aria-hidden="true"><use xlink:href="#calendar"></use></svg> ';
+    $markup .= '<span class="card-info-date event-date" itemprop="startDate" content="'. $datastart_date .'">' . esc_html($start_date) . '</span>   &nbsp;';
+    if (!empty($start_time) && !empty($end_time)){
+        $markup .= '<svg class="icon feather" aria-hidden="true"><use xlink:href="#clock"></use></svg> ';
+        $markup .= '<span class="event-time">' . esc_html($start_time) . '</span>';
+        $markup .= '<span class="event-time">' . esc_html($end_time) . '</span>';
+    } 
 } else {
-    $markup .= '<span itemprop="endDate" content="'. $dataend_date .'">' . esc_html($end_date) . '</span>';
+    $markup .= '<svg class="icon feather" aria-hidden="true"><use xlink:href="#calendar"></use></svg> ';
+    $markup .= '<span class="event-time" itemprop="startDate" content="'. $datastart_date .'">' . esc_html($start_date) . '</span>';
+    $markup .= '<span class="card-info-date event-date" itemprop="endDate" content="'. $dataend_date .'">' . esc_html($end_date) . '</span>   &nbsp;';
+    if (!empty($start_time) && !empty($end_time)){
+        $markup .= '<svg class="icon feather" aria-hidden="true"><use xlink:href="#clock"></use></svg> ';
+        $markup .= '<span>' . esc_html($start_time) . '</span>';
+        $markup .= '<span>' . esc_html($end_time) . '</span>';
+    }
 }
 
 $markup .= '<p>';
 
 if ($event->speaker !== '') {
-    $markup .= __('With', 'epfl');
-    $markup .= ': <b>' . strip_tags($event->speaker) . '</b>';
+    $markup .= '<b>' . __('Speaker', 'epfl') . ': </b>';
+    $markup .= strip_tags($event->speaker) . '';
     $markup .= '<br>';
 }
 
 if ($event->place_and_room !== '') {
-    $markup .= __('Place and room', 'epfl');
-    $markup .= ': <b><span itemprop="location">' . esc_html($event->place_and_room) . '</span></b>';
-    $markup .= '<br>';
-}
-
+    $markup .= '<b>' . __('Location', 'epfl') . ':</b>';
+    $markup .= ' <svg class="icon feather" aria-hidden="true"><use xlink:href="#map-pin"></use></svg>';
+    $markup .= ' <span itemprop="location">' . esc_html($event->place_and_room) . '</span>';
 if ($event->url_online_room !== '') {
-  $markup .= __('Online', 'epfl');
-  $markup .= ': <b>' . esc_html($event->url_online_room) . '</b>';
+        $markup .= '& <svg class="icon feather" aria-hidden="true"><use xlink:href="#monitor"></use></svg> ';
+        $markup .= 'Online';
+    }
   $markup .= '<br>';
 }
 
 if (get_current_language() == 'fr' and $event->category->fr_label !== ''){
-    $markup .= __('Category', 'epfl') . ': <b>' . esc_html($event->category->fr_label) . '</b>';
+    $markup .='<b>' . __('Category', 'epfl') . ':</b> ' . esc_html($event->category->fr_label);
```

Comment:
I would recommend to have the colon in the translation, because it's preceded by a space in french (https://fr.wikipedia.org/wiki/Deux-points) but not in English (https://en.wikipedia.org/wiki/Colon_(punctuation)).